spec/plans/discover.fmf

```diff
@@ -39,54 +39,6 @@ description: |
           description: Long test description.
           ...
 
-/shell:
-    summary: Provide a manual list of shell test cases
-    description: |
-        List of test cases to be executed can be defined manually
-        directly in the plan as a list of dictionaries containing
-        test ``name`` and actual ``test`` script. Optionally it is
-        possible to define any other :ref:`/spec/tests` attributes
-        such as ``path`` or ``duration`` here as well. The default
-        :ref:`/spec/tests/duration` for tests defined directly in
-        the discover step is ``1h``.
-
-        It is possible to fetch code from a remote git repository
-        using ``url``. In that case repository is cloned and all
-        paths are relative to the remote git root. Using remote
-        repo and local test code at the same time is not possible
-        within the same discover config, use
-        :ref:`multiple-configs` instead.
-
-        Use the :ref:`/spec/plans/discover/dist-git-source`
-        options to download rpm sources for dist-git repositories.
-
-    example:
-      - |
-        # Define several local tests
-        discover:
-            how: shell
-            tests:
-              - name: /help/main
-                test: tmt --help
-              - name: /help/test
-                test: tmt test --help
-              - name: /help/smoke
-                test: ./smoke.sh
-                path: /tests/shell
-                duration: 1m
-      - |
-        # Fetch tests from a remote repository
-        discover:
-            how: shell
-            url: https://github.com/teemtee/tmt
-            tests:
-              - name: Use tests/full/test.sh from the remote repo
-                path: /tests/full
-                test: ./test.sh
-
-    link:
-      - implemented-by: /tmt/steps/discover/shell.py
-
 /fmf:
```

Comment:
The `fmf` section should go away as well, right?

Reply:
@psss that's right, fmf removed too :)